### DISCUSSION TITLE
Unknown data point numbers

### DISCUSSION THREAD

**Discussion by shred86** [humbertogontijo/homeassistant-roborock#88] [General]
There's some payloads being received from the Roborock MQTT broker that appear to be unknown. These messages can be seen only if you have the logging level set to debugging for the integration. I'm posting them here for discussion purposes and with the hopes maybe one day we can figure out what they. Messages being received are broken up into a `data_point_number` and `data_point`.

| data_point_number  | data_point | notes
| ------------- | ------------- | ------------- |
| 121  | 22, 8 | This is being interpreted as remote control (121) but app was not being used. Observed when emptying dust bin. |
| 121  | 18, 26 | This is being interpreted as remote control (121) but app was not being used. Observed when starting a clean and mop was being washed. |
| 121  | 23 | This is being interpreted as remote control (121) but app was not being used. Observed somewhere right around mop being washed/leaving dock and returning. |
| 128 | 4, 0 | Observed when emptying dust bin. |
| 120 | 0 | Observed when washing mop. |
| 128 | 1, 3 | Observed when washing mop. |
| 128 | 1, 5 | Observed when washing mop/leaving dock. |
| 122, 126 | 88, 99, 96 | Observed when cleaning (vacuum & mop). |
| 128 | 2 | Observed when cleaning (vacuum & mop). |
| 128 | 0 | Observed when returning to dock. |

**Comment by humbertogontijo** [comment]
I've been willing to add these for a long time, the problem is that the data don't have the request id. Which is used now for responding the request
